test/specs/authorization.ts

```diff
@@ -0,0 +1 @@
+//Work in progress...
```

Comment:
@Anastas11aZakharova Please, do not add such files in the future (at least - try, idk, may be your lead require this). It just will cause you problems during the reviews with the stupid questions like "why?".

Reply:
Fixed in the latest commit